app/helpers/application_helper.rb

```diff
@@ -6,12 +6,9 @@ def pagination_info(object)
    end
   end
 
-  def display(error)
+  def inline_error(object, property)
```

Comment:
no need for this helper. simply paste in the following code in a file in your initializers directory.
```
ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
  if html_tag !~ /^\<label/
    %(#{ html_tag }
      <span class="error-msg red">
        #{ instance.error_message.join(',') }
      </span>
    ).html_safe
  else
    html_tag
  end
end
```